### DISCUSSION TITLE
Although I entered the correct password, I still encountered the  two errors

### DISCUSSION THREAD

**Discussion by dsdffgh** [keepassxreboot/keepassxc#11097] [Q&A]
1. Error while reading the database: Invalid inner header id size
2. Error while reading the database: Invalid credentials were provided, please try again.
If this reoccurs, then your database file may be corrupt. (HMAC mismatch)

**But the original file .old can be opened normally**, Who can give me some advice?

**Comment by droidmonkey** [comment]
Error one means the database file is invalid, or there was a read error if it was temporary. Error 2 may be due to same read error or you entered the wrong password. Gonna need more information about your setup now.

**Reply by dsdffgh** [reply]
Specifically, what information should I provide?

**Reply by droidmonkey** [reply]
How you are storing your database file, what you did right before this happened, etc.

**Reply by dsdffgh** [reply]
I use OneDrive to sync my password files. Maybe I opened the database file on my phone the day before, but I don't remember doing anything else.

**Reply by droidmonkey** [reply]
Is this still a problem or only temporary?

**Reply by dsdffgh** [reply]
yes, the problem still exists
